launch/launch/__init__.py

```diff
@@ -47,6 +51,9 @@ class TaskDescriptor(object):
     def __init__(self):
         self.task_state = None
 
+    def send_signal(self, signal):
+        pass
+
```

Comment:
Since the other subclass `CoroutineDescriptor` can't implement this the base class should probably not contain this method.